### PR TITLE
Added method for getting channel key or conversation ID

### PR DESCRIPTION
**Pull request by AbhijeetRanjan308**
## Summary 
- Added a method to get channel key if you pass conversation ID or conversation ID if you pass channel key

## Function
```
/// Just need to pass either 'clientChannelKey' or 'channelID' 
dynamic objc = {'clientChannelKey': 'pKC6riItFm7eMmogrJ2gz3LvjrD9nwaU', 'channelID': 95713948}   

KommunicateFlutterPlugin.getConversarionIdOrKey(objc).then((value) {
                     print('User Channel id : $value');
                    }
                  ).catchError((error) {
                      print('Error in channel user details : ' +
                          error.toString());
                    });

### CODE REVIEW

File: android/src/main/java/io/kommunicate/kommunicate_flutter_plugin/KmMethodHandler.java [hunk 1]
```diff
@@ -156,6 +156,50 @@ public void onFailure(Object error) {
                     result.error(ERROR, error.toString(), null);
                 }
             });
+        } else if (call.method.equals("getConversarionIdOrKey")) {
+            try {
+            JSONObject searchObject = new JSONObject(call.arguments.toString());
```
Comment (Sathyan-KM) [inline android/src/main/java/io/kommunicate/kommunicate_flutter_plugin/KmMethodHandler.java:161]: rename the variable

File: ios/Classes/SwiftKommunicateFlutterPlugin.swift [hunk 2]
```diff
@@ -145,11 +145,44 @@ public class SwiftKommunicateFlutterPlugin: NSObject, FlutterPlugin, KMPreChatFo
...
+                    }
+                    self.sendSuccessResult(message: channel.clientChannelKey)
+                }
+            } else if let clientChannelKey = jsonObj["clientChannelKey"] {
+                guard let clientChannelKey = clientChannelKey as? String else {
+                    self.sendErrorResultWithCallback(result: result, message: "Client Channel Key is not String")
+                    return
+                }
+                alChannelService.getChannelInformation(nil, orClientChannelKey: clientChannelKey) { channel in
+                    guard let channel = channel , let channelKey = channel.key else {
```
Comment (Sathyan-KM) [inline ios/Classes/SwiftKommunicateFlutterPlugin.swift:176]: rename it to conversation id or something.

File: ios/Classes/SwiftKommunicateFlutterPlugin.swift [hunk 3]
```diff
@@ -145,11 +145,44 @@ public class SwiftKommunicateFlutterPlugin: NSObject, FlutterPlugin, KMPreChatFo
...
+                        self.sendErrorResultWithCallback(result: result, message: "Conversation Not Found \(channel)")
+                        return
+                    }
+                    self.sendSuccessResult(message: "\(channelKey)")
+                }
+            } else {
+                self.sendErrorResultWithCallback(result: result, message: "Wrong Object passed for searching conversation information")
```
Comment (Sathyan-KM) [inline ios/Classes/SwiftKommunicateFlutterPlugin.swift:183]: error message is not contextual

File: lib/kommunicate_flutter.dart [hunk 4]
```diff
@@ -51,6 +51,10 @@ class KommunicateFlutterPlugin {
     return await _channel.invokeMethod('sendMessage', jsonEncode(messageObjc));
   }
 
+  static Future<dynamic> getConversarionIdOrKey(dynamic searchObj) async {
```
Comment (Sathyan-KM) [inline lib/kommunicate_flutter.dart:54]: rename the seachobj variable

**Review by Sathyan-KM — CHANGES_REQUESTED** [review]

File: lib/kommunicate_flutter.dart (hunk 4 above)
Comment (AbhijeetRanjan308) [reply lib/kommunicate_flutter.dart:54]: renamed it to 'conversationIdObjc'

File: ios/Classes/SwiftKommunicateFlutterPlugin.swift (hunk 3 above)
Comment (AbhijeetRanjan308) [reply ios/Classes/SwiftKommunicateFlutterPlugin.swift:183]: error message ""Object doesn't contain 'clientChannelKey' or 'channelID'""

File: android/src/main/java/io/kommunicate/kommunicate_flutter_plugin/KmMethodHandler.java (hunk 1 above)
Comment (AbhijeetRanjan308) [reply android/src/main/java/io/kommunicate/kommunicate_flutter_plugin/KmMethodHandler.java:161]: renamed the variable to 'conversationIdObjc'

File: ios/Classes/SwiftKommunicateFlutterPlugin.swift (hunk 2 above)
Comment (AbhijeetRanjan308) [reply ios/Classes/SwiftKommunicateFlutterPlugin.swift]: renamed the variable to 'conversationID'

**Review by pranaysaini04 — APPROVED** [review]

**Pull request merged**